cmd/lotus-bench/amt_internal.go

```diff
@@ -0,0 +1,309 @@
+package main
+
```

Comment:
This is just copy pasted from AMT package. Couldnt import because it is internal over there.

Reply:
Can you explain/link directly to the AMT package?

Reply:
And this is just code for reading/writing from go memory structures to cbor bytes

Reply:
I think the rest of the code looks great, just this question: Does it make sense to add a comment to the top of the file with this link and maybe an explanation of why you are adding it to this repo instead of linking/import it?

Reply:
Also there's a lint issue